app/services/customers/manage_invoice_custom_sections_service.rb

```diff
@@ -35,14 +38,22 @@ def call
 
     private
 
-    attr_reader :customer, :section_ids, :skip_invoice_custom_sections
+    attr_reader :customer, :section_ids, :skip_invoice_custom_sections, :section_codes
+
+    def raise_double_selection
```

Comment:
Not sure of the naming here as `result.validation_failure!` is not raising an error but only marking the result object as failed and adding the error detail to it.

Reply:
🙌